source/includes/usage-examples/sample-app-intro.rst

```diff
@@ -0,0 +1,9 @@
+Sample Application
+~~~~~~~~~~~~~~~~~~
+
+You can use the following sample application to test the code examples on this
+page. To use the sample application, perform the following steps:
+
+1. Ensure you have the {+driver-short+} installed in your Maven or Gradle project.
```

Comment:
S: maybe we should have a tip box somewhere linking to the future QS page to give installation directions

Reply:
Can add a commented-out tip box until said QS page exists.